db/migrate/20161024182548_create_questions.rb

```diff
@@ -0,0 +1,25 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    create_table :questions do |t|
+      t.text            :text
+      t.timestamps null: false
+    end
+
+    create_table :response_choices do |t|
+      t.belongs_to      :questions, index: true
```

Comment:
Immediately after saying that we paired on editing an old migration 😂 Next time we take the high road!